### ISSUE TITLE
Lancer un appel à dev+designers sur le discord Astro

### ISSUE THREAD

**Issue by lenaMK** [ouvroir/labouvroir#86]
Format à compléter au mieux

💼 **Job Title**
Relevant country flag (e.g.:flag-cm: or:flag-mx:).

💸 **Salary Range**
This is required and we might remove posts without one!

🏢 🌴 🏠 **Remote Status** 
For example: office, fully remote, work from anywhere, work from a particular country.

📎 **Job Description**
Include a thorough description of this role.

🍵  **Company Info** 
Share more about the company culture, team, and company info.

📧 **Contact Info** 
Please include the point of contact for the role and a link to appl

**Comment by emchateau** [comment]
C’est compliqué car c’est plutôt un emploi qu’une offre en free lance. Est-ce qu’il y a des annonces free lance ?

**Comment by lenaMK** [comment]
non, ils ont dit de simplement préciser que c'est une offre freelance et donner un max d'information à partir de ce template

**Comment by emchateau** [comment]
Alors OK.
Canada / Québec
Free lance
Remote, work from anywhere in Québec or Canada

Développer Astro/Svelte pour la conception d’un site web d’un projet de recherche en sciences humaines en lien avec la muséologie. Construction d’une coquille pour la création de programmes de conférences.

Quelques mots sur le Partenariat

Qu’en dis-tu ?

**Comment by lenaMK** [comment]
Salary range est-ce qu'on peut donner une fouchette du taux à l'heure (selon expérience) ou une autre indication?
